packages/connect-core/src/entities/ForwardingPath.ts

```diff
@@ -0,0 +1,68 @@
+import { providers as ethersProviders } from 'ethers'
+
+import { buildApprovePretransaction } from '../utils/transactions'
+import {
+  ForwardingPathData,
+  AppOrAddress,
+  StepDescribed,
+  TokenData,
+} from '../types'
+import ForwardingPathDescription, {
+  describePath,
+} from '../utils/descriptor/index'
+import App from './App'
+import Transaction from './Transaction'
+
+export default class ForwardingPath {
+  #apps: App[]
+  #provider: ethersProviders.Provider
+  readonly destination: AppOrAddress
+  readonly transactions: Transaction[]
+
+  constructor(
```

Comment:
I finally decided to construct a `ForwardingPath` with the list of `Transactions` and the destination. This is relevant as the description of a forwarding path happened as a second step when we call `describe()` method.